proto/sifnode/clp/v1/params.proto

```diff
@@ -6,4 +6,16 @@ import "gogoproto/gogo.proto";
 option go_package = "github.com/Sifchain/sifnode/x/clp/types";
 
 // Params - used for initializing default parameter for clp at genesis
-message Params { uint64 min_create_pool_threshold = 1; }
+message Params {
+  uint64 min_create_pool_threshold = 1;
+  uint64 liquidity_removal_lock_period = 2; // in blocks
+  repeated RewardPeriod reward_periods = 3;
+}
+
+message RewardPeriod {
+  uint64 start_block = 1;
+  uint64 end_block = 2;
+  string allocation = 3 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Uint"
+  ];
+}
```

Comment:
@timlind @Jedi2002 Do you guys think we should use a type for PMTP period params also? Looks like this is here to allow for multiple periods at once but it may be good to do on PMTP also to keep things uniform?
https://github.com/Sifchain/sifnode/pull/2507/files